src/gta/script_handler.hpp

```diff
@@ -93,16 +93,17 @@ class CGameScriptHandlerNetwork : public CGameScriptHandler
 	uint8_t m_0xAF;       // 0xAF
 };
 
-class CScriptParticipant
+#pragma pack(push, 8)
+struct CScriptParticipant
```

Comment:
This should probably be moved to the GTAV-Classes project. Rule of thumb I've always imagined, is if it comes from within GTA V itself, it should be apart of GTAV-Classes.